tests/licensedcode/test_plugin_license.py

```diff
@@ -242,6 +242,24 @@ def test_reindex_licenses_works():
     run_scan_click(['--reindex-licenses'])
 
 
+def test_detection_with_external_license_directories():
+    test_dir = test_env.get_test_loc('plugin_license/external_licenses/scan', copy=True)
+    external_license_dir = f"{os.path.join(os.path.dirname(__file__), 'data')}/example_external_licenses/"
```

Comment:
You can use `test_env.get_testloc()` to get your test directory location since we already set up that stuff on line 22.
```suggestion
    external_license_dir = test_env.get_test_loc('example_external_licenses/example1')
```